<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GXPMainWindow</class>
 <widget class="QMainWindow" name="GXPMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GXParser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../gxparser.qrc">
    <normaloff>:/icons/images/euracelogo.png</normaloff>:/icons/images/euracelogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Model File Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filePathLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="chooseFilePathButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="xparserButton">
        <property name="text">
         <string>Run XParser</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gnuMakeButton">
        <property name="text">
         <string>Run GNU Make</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>XParser Output Messages</string>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QTextEdit" name="xparserOutputTextEdit"/>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Directory Layout (Compilation Output)</string>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QTreeView" name="filesTreeView"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionChoose_Model_File"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChoose_Model_File">
   <property name="text">
    <string>Ch&amp;oose Model File</string>
   </property>
  </action>
  <action name="actionSave_Compilation_Output">
   <property name="text">
    <string>Save Compilation Output</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../gxparser.qrc"/>
 </resources>
 <connections/>
</ui>
